### PR TITLE
Added CONTRIBUTING and CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by kirantejatummuri**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Hard work paid off
+
+Congratulations on succesfully contributing and the whole community thanks you!!!!
+
+Once you PR is merged, you will be added to the contributors list.
+
+
+## Need more help?
+
+If for any reason you have questions on the process, feel free to reach us back at [igupta@ncsu.edu](mailto:ktummur@ncsu.edu). Github process can be very complex and we dont want to loose your valuable contributions because of that. We are very excited that you have visited us and you will make our project much better.
```
Comment (isha-bansal0115) [inline CONTRIBUTING.md]: in [], it has my email id and in () it has yours. Can you proof read this once?

**Review by isha-bansal0115 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kirantejatummuri) [reply CONTRIBUTING.md]: Changed it, thank you!

**Comment by rahulkalita8** [comment]
@isha-bansal0115 I did not review it. Does it look good to you. If yes, please let me know. I will merge

**Review by isha-bansal0115 — APPROVED** [review]
Looks good.

@kirantejatummuri your PR has conflicts. Please resolve and update so that we can merge

**Comment by isha-bansal0115** [comment]
@rahulkalita8  looks good. You can merge

**Pull request merged**